README.md

```diff
@@ -62,25 +63,22 @@ datadog.addForwarderToNonLambdaLogGroups([<LOG_GROUPS>])
-| `apiKmsKey`             | `api_kms_key`             | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set, and you are using KMS encryption.                                                                                                                                                                     |
-| `enableDatadogTracing`  | `enable_datadog_tracing`  | Enable Datadog tracing on your Lambda functions. Defaults to `true` .                                                                                                                                                                                                                                                                                      |
-| `injectLogContext`      | `inject_log_context`      | When set, the lambda layer will automatically patch console.log with Datadog's tracing ids. Defaults to `true` .                                                                                                                                                                                                                                           |
-
-### AWS CDK Configurations
-
-The AWS CDK library offers some additional options that you may include in your lambda functions and application / stack.
+| `forwarderArn` | `forwarder_arn` | When set, the plugin will automatically subscribe the Datadog Forwarder to the functions' log groups. Do not set `forwarderArn` when `extensionLayerVersion` is set. |
+| `flushMetricsToLogs` | `flush_metrics_to_logs` | Send custom metrics using CloudWatch logs with the Datadog Forwarder Lambda function (recommended). Defaults to `true` . If you disable this parameter, it's required to set `apiKey` (or `apiKmsKey` ). |
+| `site` | `site` | Set which Datadog site to send data. This is only used when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set. Possible values are `datadoghq.com` , `datadoghq.eu` , `us3.datadoghq.com` and `ddog-gov.com`. The default is `datadoghq.com`. |
+| `apiKey` | `api_key` | Datadog API Key, only needed when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set. For more information about getting a Datadog API key, see the [API key documentation][8]. |
+| `apiKmsKey` | `api_kms_key` | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set, and you are using KMS encryption. |
+| `enableDatadogTracing` | `enable_datadog_tracing` | Enable Datadog tracing on your Lambda functions. Defaults to `true`. |
+| `injectLogContext` | `inject_log_context` | When set, the Lambda layer will automatically patch console.log with Datadog's tracing ids. Defaults to `true`. |
+| `logLevel` | `log_level` | When set to `debug`, the Datadog Lambda Library and Extension will log additional information to help troubleshoot issues. |
```

Comment:
This `logLevel` entry is the only new text in the table -- for the rest of the table I just removed a bunch of extra whitespace and hyphens.

Reply:
I personally find the extra whitespace and hyphens extremely pleasing when working with the README in a text editor on an  arbitrary width monitor.
![image](https://user-images.githubusercontent.com/1136297/123321961-a67fa980-d501-11eb-9415-2a1855f3750b.png)

(do what thou wilt, though)

Reply:
Unfortunately it doesn't work so great on a laptop screen, or comparing a diff side-by-side. Also, the whitespace and hyphens need to be updated as the lengths of the different columns change.